internal/transport/rest/handler.go

```diff
@@ -0,0 +1,211 @@
+package rest
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/Viquad/crud-app/internal/domain"
+	"github.com/julienschmidt/httprouter"
+)
+
+type Handler struct {
+	accountService domain.AccountService
```

Comment:
services *service.Services

Reply:
@tarkovskynik So, you decided it is better to remove interfaces `domain.Services` and `domain.Repositories`. In this case dependencies `rest`<-`service` and `service`<-`repository` are neccessary to pass `service.Services` as parameter for `rest.handler` and `repository.Repositories` as parameter in `service.Services`. Maybe it would be better to move `domain.Services` to `rest.Services` and `domain.Repositories` to `service.Repositories`. This will help to avoid dependencies and locate interfaces where they using as Maxim Zhashkevich suggests.

Reply:
@tarkovskynik [here](https://github.com/Viquad/crud-app/pull/2/commits/de973c4df3364311a3c37d8ed1380dbd5eccb40d) is how I see it.